src/biome/text/dataset.py

```diff
@@ -7,10 +7,15 @@
 from typing import Union, Dict, Iterable, List, Any, Tuple, Optional, TYPE_CHECKING
 
 import datasets
+from datasets.fingerprint import Hasher
 from allennlp.data import AllennlpDataset, AllennlpLazyDataset, Instance
+from biome.text.about import __version__ as biome__version__
```

Comment:
`__version__` is already enabled from `biome.text` package

```suggestion
from biome.text import __version__ as biome__version__
```

Reply:
Just figured out that these are not the same ... `from biome.text import __version__` is the biome.text version of the last install (basically when `make dev` was executed when developing) and `from biome.text.about import __version__` is the biome.text version of the last commit. I think i prefer latter one, which seems more convenient for development.

Reply:
Naaa, this does not work!! `from biome.text.about import __version__` depends on the current working directory!! We have to use `from biome.text import __version__` and keep this in mind when developing.